.gitignore

```diff
@@ -2,7 +2,6 @@
 *.sw*
 
 .vscode
-*.cmake
```

Comment:
CMake creates files named `cmake_install.cmake` that we don't want included. Perhaps change the ignore to that file specifically?